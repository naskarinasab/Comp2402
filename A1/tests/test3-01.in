magnet
house
white out
television
conditioner
seat belt
remote
carrots
bookmark
wallet
candle
thermostat
socks
book
sun glasses
vase
pool stick
blanket
lip gloss
door
soap
clock
CD
bow
bread
flowers
fridge
clothes
grid paper
sailboat
toe ring
shovel
chalk
eraser
cell phone
sticky note
food
thermometer
tv
USB drive
bottle cap
spring
outlet
drawer
balloon
fork
cookie jar
coasters
hair brush
sand paper
fake flowers
soda can
shirt
plastic fork
washing machine
couch
car
controller
lamp shade
cup
charger
plate



screw
shawl
shampoo
knife
glass
bananas
bed
teddies
bottle


























mouse pad
