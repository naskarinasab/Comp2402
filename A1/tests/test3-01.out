soap
clock
CD
chalk
eraser
cell phone
sticky note
food
thermometer
tv
USB drive
bottle cap
spring
outlet
drawer
balloon
fork
cookie jar
coasters
hair brush
sand paper
fake flowers
soda can
shirt
plastic fork
washing machine
couch
car
controller
